pkg/config/tlscfg/testdata/README.md

```diff
@@ -1,35 +0,0 @@
-# Example Certificate Authority and Certificate creation for testing
```

Comment:
I would keep the README with some basic instruction how to regenerate certificates, preferably via a make command.

Reply:
README.md and Makefile added.

Reply:
I was actually referring to the main Makefile. If it's just in this dir then I would simply provide instructions for running the script directly.